lib/suspenders/app_builder.rb

```diff
@@ -113,10 +113,6 @@ def set_up_hound
       copy_file "hound.yml", ".hound.yml"
     end
 
-    def configure_newrelic
```

Comment:
I've found it useful to find issues in staging and production. Why would you not use it in staging? Notifications / priority? Too much noise?